third_party/move/move-compiler-v2/src/experiments.rs

```diff
@@ -79,6 +79,11 @@ pub static EXPERIMENTS: Lazy<BTreeMap<String, Experiment>> = Lazy::new(|| {
             description: "Turns on or off access and use checks".to_string(),
             default: Inherited(Experiment::CHECKS.to_string()),
         },
+        Experiment {
+            name: Experiment::ACQUIRES_CHECK.to_string(),
+            description: "Turns on or off v1 style acquires checks".to_string(),
+            default: Given(false),
```

Comment:
Default should be on. Then turn it off for the access control transactional tests.

Reply:
done